packages/gatsby-plugin-theme-ui/README.md

```diff
@@ -16,6 +16,31 @@ module.exports = {
 In addition to providing context, this plugin will also
 prevent a flash of unstyled colors when using color modes.
 
+## Options
+
+| Key                      | Default value    | Description                                                                      |
+| ------------------------ | ---------------- | -------------------------------------------------------------------------------- |
+| `themePreset`               | `null`            | This can be a JSON theme object or a string package name. Make sure the package you're requiring is installed in your dependencies.               |
```

Comment:
NB (and not to bikeshed), but what are your thoughts on making this simply `preset`? Generally trying to avoid overuse of the word *theme* if possible :)

Reply:
Bikeshedding is fair! I actually debated this quite a bit. My fear is twofold. One, if you pass it from a site using `blog-theme` for example you lose context. And two is that `prismPreset` is also a preset and therefore a bit confusing?

Reply:
I think for Theme UI, the `theme` object is the main one and Prism is a secondary/optional library -- for the blog theme, maybe the theme's options are nested as `options.themeUIOptions`, which seems like a convention used some of the time, *or* we could map something like `options.themeUIPreset` to this `options.preset` in the blog theme

Reply:
That's fair. Updated.